scripts/variantstore/wdl/ngs_filter_extract.wdl

```diff
@@ -0,0 +1,497 @@
+        File omni_resource_vcf_index
+        File one_thousand_genomes_resource_vcf
+        File one_thousand_genomes_resource_vcf_index
+        File mills_resource_vcf
+        File mills_resource_vcf_index
+        File axiomPoly_resource_vcf
+        File axiomPoly_resource_vcf_index
+        File dbsnp_resource_vcf = dbsnp_vcf
+        File dbsnp_resource_vcf_index = dbsnp_vcf_index
+
+        # Runtime attributes
+        Int? small_disk_override
+        Int small_disk = select_first([small_disk_override, "100"])
+        Int? medium_disk_override
+        Int medium_disk = select_first([medium_disk_override, "200"])
+        Int? large_disk_override
+        Int large_disk = select_first([large_disk_override, "300"])
+        Int? huge_disk_override
+        Int huge_disk = select_first([huge_disk_override, "400"])
+
+        String? preemptible_tries_override
+        Int preemptible_tries = select_first([preemptible_tries_override, "3"])
+
+    }
+    
+    scatter(i in range(max_chrom_id)) {
+        call ExtractFilterTask {
+            input:
+                gatk_override            = gatk_override,
+                reference                = reference,
+                reference_index          = reference_index,
+                reference_dict           = reference_dict,
+                fq_sample_table          = fq_sample_table,
+                chrom_offset             = chrom_offset,
+                chrom_id                 = i+1,
+                fq_alt_allele_table      = fq_alt_allele_table,
+                read_project_id          = query_project,
+                output_file              = "${output_file_base_name}_${i}.vcf.gz"
+        }
+    }
+
+    call MergeVCFs { 
+       input:
+           input_vcfs = ExtractFilterTask.output_vcf,
+           input_vcfs_indexes = ExtractFilterTask.output_vcf_index,
+           output_vcf_name = "${output_file_base_name}.vcf.gz",
+           preemptible_tries = 3
+    }
+    
+    call IndelsVariantRecalibrator {
+        input:
+        sites_only_variant_filtered_vcf = MergeVCFs.output_vcf,
+        sites_only_variant_filtered_vcf_index = MergeVCFs.output_vcf_index,
+        recalibration_filename = filter_set_name + ".indels.recal",
+        tranches_filename = filter_set_name + ".indels.tranches",
+        recalibration_tranche_values = indel_recalibration_tranche_values,
+        recalibration_annotation_values = indel_recalibration_annotation_values,
+        mills_resource_vcf = mills_resource_vcf,
+        mills_resource_vcf_index = mills_resource_vcf_index,
+        axiomPoly_resource_vcf = axiomPoly_resource_vcf,
+        axiomPoly_resource_vcf_index = axiomPoly_resource_vcf_index,
+        dbsnp_resource_vcf = dbsnp_resource_vcf,
+        dbsnp_resource_vcf_index = dbsnp_resource_vcf_index,
+        use_allele_specific_annotations = true,
+        disk_size = large_disk
+    }
+
+    call SNPsVariantRecalibrator as SNPsVariantRecalibratorClassic {
```

Comment:
nit - we could remove the as here (i'm guessing we used to have a classic and non classic mode)

Reply:
yeah -- I think we're going to have a "non-classic" mode as well once we start scale testing.  The WARP pipeline has it for > 10k wgs samples... but it's a bunch of logic.  Happy to remove the "as ..." for now though